crates/dekaf/src/task_manager.rs

```diff
@@ -576,6 +649,60 @@ pub enum DekafTaskAuth {
     },
 }
 
+impl DekafTaskAuth {
+    fn exp(&self) -> u64 {
+        match self {
+            DekafTaskAuth::Redirect { fetched_at, .. } => {
+                // Redirects are valid for 10 minutes
```

Comment:
I just made this up. Since redirects don't get a token, they don't get an `agent-api`-specified expiration.

Reply:
🤔 Does this mean that we'll now start to expire `Redirect`s, where previously we haven't? Just wanting to double check whether that poses any risk for existing tasks that may not have encountered that before.